docs/source/en/model_doc/code_llama.md

```diff
@@ -49,6 +49,8 @@ Here is a sample usage
 python src/transformers/models/llama/convert_llama_weights_to_hf.py \
     --input_dir /path/to/downloaded/llama/weights --model_size 7B --output_dir /output/path
 ```
+Note that executing the script requires enough CPU RAM to host the whole model in float16 precision (even if the biggest versions
+come in several checkpoints they each contain a part of each weight of the model, so we need to load them all in RAM).
```

Comment:
It could be an option to lightly promote https://huggingface.co/spaces/hf-accelerate/model-memory-usage here cc @muellerzr :grin:

Reply:
That'd be awesome if so! Something like:

Note that executing the script requires enough CPU RAM to host the whole model in float16 precision (even if the biggest versions come in several checkpoints they each contain a part of each weight of the model, so we need to load them all in RAM). To see how much CPU and GPU memory you need for this model or others, try this calculator which can help determine that value.